src/index.js

```diff
@@ -1,128 +1,93 @@
-import _ from 'lodash';
+import { navBar } from './navBar';
+import { footer } from './footer';
+import { home } from './home';
+import { menu } from './menu';
+import { about } from './about';
+import { contact } from './contact';
 
 function component() {
```

Comment:
Use es6+ throughout your app ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/restaurant_page/es6_requirement.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59622983/)).
```suggestion
const component = () => {
```